### PR TITLE
Add description

### LINKED ISSUE #89: Add task description

### PR DESCRIPTION
**Pull request by kiatkat**
Add a `description` command that allows the user to add a description to a task.

Resolves #89.

### CODE REVIEW

File: src/main/java/profplan/logic/commands/DescriptionCommand.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package profplan.logic.commands;
```
Comment (eyelessrhyme7) [inline src/main/java/profplan/logic/commands/DescriptionCommand.java:1]: Great job on creating a new command and not relying on existing commands like add/edit. Brings more uniqueness to our product.

File: src/main/java/profplan/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -23,23 +24,29 @@ public class SampleDataUtil {
     public static Task[] getSampleTasks() {
         return new Task[] {
             new Task(new Name("Alex Yeoh"), new Priority("4"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
```
Comment (eyelessrhyme7) [inline src/main/java/profplan/model/util/SampleDataUtil.java:26]: Good job changing by-products to reflect change in Task instance.

**Review by eyelessrhyme7 — APPROVED** [review]

**Review by jack1e0 — APPROVED** [review]
lgtm

**Review by newway1814 — COMMENTED** [review]
LGTM!

**Pull request merged**